corehq/apps/users/static/users/js/mobile_workers.ng.js

```diff
@@ -379,6 +379,13 @@
                 ctrl.$validators.validatePassword = function (password) {
                     if (!password) {
                         return false;
+                    } else if (!(
+                        password.length >= 8 &&
+                        /\W/.test(password) &&
+                        /\d/.test(password) &&
+                        /[A-Z]/.test(password)
```

Comment:
just uppercase according to https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/forms.py#L1210-L1222 probably just assume that there are lower case in the password as a default